text/0033-private-network-guide.md

```diff
@@ -52,18 +50,15 @@ Here is a table summarizing the minimum requirements for each type of instance:
 |---  | ---:  | ---: | ---: | ---:     | --- |
 |Full node         | 3 | 4 | 8 GB | 4 GB | hathornetwork/hathor-core
 |Wallet            | 1 | 2 | 2 GB | 100 MB | hathornetwork/hathor-wallet-headless
-|Miner             | 2 | 2 | 1 GB | 100 MB | hathornetwork/cpuminer
```

Comment:
We will still have a miner connected to the tx-mining-service. Maybe we should keep this line.

Reply:
OK: change made: 
Readded the line as suggested.